.github/CODEOWNERS

```diff
@@ -522,29 +525,33 @@ x-pack/examples/files_example @elastic/kibana-app-services
 /x-pack/plugins/security_solution/server/lib/detection_engine/routes/index @elastic/security-detections-response-alerts
 /x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals @elastic/security-detections-response-alerts
 
-
 ## Security Solution sub teams - Detections and Response Rules
-/x-pack/plugins/security_solution/cypress/integration/detection_rules @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/rule_monitoring @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/schemas/common @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/schemas/request @elastic/security-detections-response-rules
+/x-pack/plugins/security_solution/common/detection_engine/schemas/response @elastic/security-detections-response-rules
```

Comment:
@marshallmain Do you want me to add the Alerts area to these folders as well?

Reply:
I think these should belong to the rules area, from the alerts area I contribute to them only because I had already worked on them a lot (but didn't finish :cry: ) prior to splitting into area teams.

Reply:
Sounds good to me!